### PR TITLE
fixed deps for animation and nested animation examples

### PR DESCRIPTION
**Pull request by deowk**
Just fixed the animation examples to reference the correct dependency.

### CODE REVIEW

**Comment by knowbody** [comment]
thanks for that, just fixed the `package.json` remove what you've added to `dependencies` and just fix the `devDependencies`. We don't want to have `react` and CSSTransitionGroup in `dependencies`

**Comment by deowk** [comment]
@knowbody yeah right, sorry about that - fixed...;)

File: package.json [hunk 1]
```diff
@@ -58,6 +58,7 @@
     "react": "0.14.0",
     "react-addons-test-utils": "0.14.0",
     "react-addons-transition-group": "0.14.0",
```
Comment (knowbody) [inline package.json]: @deowk and this line should be deleted

**Comment by knowbody** [comment]
thanks!

**Pull request merged**